functionary/core/models/function.py

```diff
@@ -43,6 +44,13 @@ class Function(models.Model):
     variables = models.JSONField(default=list, validators=[list_of_strings])
     return_type = models.CharField(max_length=64, null=True)
     schema = models.JSONField()
+    active = models.BooleanField(default=True)
+
+    def deactivate(self):
```

Comment:
Move this method to the bottom, below `__str__`.

Reply:
Fixed.